src/main/app/windowManager.js

```diff
@@ -160,29 +158,60 @@ class WindowManager extends EventEmitter {
   }
 
   /**
-   * Returns the active window id or null if no window is registred.
-   * @returns {number|null}
+   * Returns the active window or null if no window is registered.
+   * @returns {BaseWindow|undefined}
    */
   getActiveWindow () {
+    return this._windows.get(this._activeWindowId)
+  }
+
+  /**
+   * Returns the active window id or null if no window is registered.
+   * @returns {number|null}
+   */
+  getActiveWindowId () {
     return this._activeWindowId
   }
 
-  get windows () {
-    return this._windows
+  /**
+   * Returns the (last) active editor window or null if no editor is registered.
+   * @returns {EditorWindow|undefined}
+   */
+  getActiveEditor () {
+    let win = this.getActiveWindow()
+    if (win && win.type !== WindowType.EDITOR) {
+      win = this._windows.get(this._windowActivity.getSecondNewest())
+      if (win && win.type === WindowType.EDITOR) {
+        return win
+      }
+      return undefined
+    }
+    return win
   }
 
-  get windowCount () {
-    return this._windows.size
+  /**
+   * Returns the (last) active editor window id or null if no editor is registered.
+   * @returns {number|null}
+   */
+  getActiveEditorId () {
```

Comment:
Why not call `getActiveEditor` and get the active editor id?

Reply:
Well, I don't know 😄 I'll change this...